### PR TITLE
GG-29309 Prepare assembly for Control Center Agent in GridGain

### PR DESCRIPTION
**Pull request by vveider**

### CODE REVIEW

File: assembly/dependencies-apache-ignite-lgpl.xml [hunk 1]
```diff
@@ -185,4 +186,26 @@
...
         </moduleSet>
     </moduleSets>
+
+    <fileSets>
+        <fileSet>
+            <directory>${basedir}/../control-center-agent/modules/control-center-agent/target</directory>
+            <outputDirectory>optional/control-center-agent</outputDirectory>
+            <includes>
+                <include>gridgain-control-center-agent-*.jar</include>
+            </includes>
```
Comment (nva) [inline assembly/dependencies-apache-ignite-lgpl.xml:196]:
```suggestion
            <includes>
                <include>*.jar</include>
            </includes>
            <excludes>		
                <exclude>*-tests.jar</exclude>		
                <exclude>*-javadoc.jar</exclude>		
                <exclude>*-sources.jar</exclude>		
            </excludes>
```

File: assembly/dependencies-apache-ignite-lgpl.xml [hunk 2]
```diff
@@ -141,6 +141,7 @@
                 <exclude>org.gridgain:ignite-ml-spark-model-parser</exclude>
                 <exclude>org.gridgain:ignite-ml-tensorflow-model-parser</exclude>
                 <exclude>org.gridgain:ignite-ml-xgboost-model-parser</exclude>
+                <exclude>org.gridgain:ignite-management-console-agent</exclude>
```
Comment (nva) [inline assembly/dependencies-apache-ignite-lgpl.xml:144]: @akuznetsov-gridgain We should remove this in GG-29327

File: assembly/dependencies-apache-ignite.xml [hunk 3]
```diff
@@ -191,4 +192,26 @@
...
         </moduleSet>
     </moduleSets>
+
+    <fileSets>
+        <fileSet>
+            <directory>${basedir}/../control-center-agent/modules/control-center-agent/target</directory>
+            <outputDirectory>optional/control-center-agent</outputDirectory>
+            <includes>
+                <include>gridgain-control-center-agent-*.jar</include>
+            </includes>
```
Comment (nva) [inline assembly/dependencies-apache-ignite.xml:202]:
```suggestion
            <includes>
                <include>*.jar</include>
            </includes>
            <excludes>		
                <exclude>*-tests.jar</exclude>		
                <exclude>*-javadoc.jar</exclude>		
                <exclude>*-sources.jar</exclude>		
            </excludes>
```

File: assembly/dependencies-apache-ignite-lgpl.xml (hunk 2 above)
Comment (vveider) [reply assembly/dependencies-apache-ignite-lgpl.xml:144]: I agree, but then the change should also be done in GG-29327.

Comment (akuznetsov-gridgain) [reply assembly/dependencies-apache-ignite-lgpl.xml:144]: I've already prepared PR fro GG-29327
we can merge branches if needed

**Pull request merged**